diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 0c78d0e388..a4bb508d9b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -416,6 +416,7 @@ static_library("browser") {
     "//chrome/browser/profiles:profile_util_impl",
     "//chrome/browser/profiles:profile_manager",
     "//seoul/browser/organization",
+    "//seoul/browser/lifecycle",
     "//chrome/browser/profiles:profile_manager_impl",
 
     # TODO(crbug.com/353332589): Remove this circular dependency once
