diff --git a/chrome/browser/ui/tabs/features.cc b/chrome/browser/ui/tabs/features.cc
index f99a2eb237..9bdcaba5f6 100644
--- a/chrome/browser/ui/tabs/features.cc
+++ b/chrome/browser/ui/tabs/features.cc
@@ -15,7 +15,7 @@ BASE_FEATURE(kTabGroupHome, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kSessionRestoreShowThrobberOnVisible,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kVerticalTabs, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(kVerticalTabs, base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kVerticalTabsLaunch, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE_PARAM(bool,
diff --git a/chrome/browser/ui/tabs/tab_strip_prefs.cc b/chrome/browser/ui/tabs/tab_strip_prefs.cc
index c0d39dedda..5e65ffb5cd 100644
--- a/chrome/browser/ui/tabs/tab_strip_prefs.cc
+++ b/chrome/browser/ui/tabs/tab_strip_prefs.cc
@@ -46,11 +46,11 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
       prefs::kTabSearchPinnedToTabstripMigrationComplete2, false);
   registry->RegisterBooleanPref(prefs::kProjectsPanelPinnedToTabstrip, true);
   registry->RegisterBooleanPref(prefs::kEverythingMenuPinnedToTabstrip, true);
-  registry->RegisterBooleanPref(prefs::kVerticalTabsEnabled, false);
+  registry->RegisterBooleanPref(prefs::kVerticalTabsEnabled, true);
   registry->RegisterBooleanPref(
       prefs::kVerticalTabsExpandOnHoverEnabled,
       tabs::kVerticalTabsExpandOnHoverDefaultEnabled.Get());
-  registry->RegisterBooleanPref(prefs::kVerticalTabsEnabledFirstTime, false);
+  registry->RegisterBooleanPref(prefs::kVerticalTabsEnabledFirstTime, true);
   registry->RegisterBooleanPref(prefs::kVerticalTabsCollapsedState, false);
   registry->RegisterIntegerPref(prefs::kVerticalTabsUncollapsedWidth,
                                 kVerticalTabStripDefaultUncollapsedWidth);
diff --git a/chrome/browser/ui/tabs/vertical_tab_strip_state_controller_unittest.cc b/chrome/browser/ui/tabs/vertical_tab_strip_state_controller_unittest.cc
index 88981f5e91..79cc777c43 100644
--- a/chrome/browser/ui/tabs/vertical_tab_strip_state_controller_unittest.cc
+++ b/chrome/browser/ui/tabs/vertical_tab_strip_state_controller_unittest.cc
@@ -76,7 +76,7 @@ class VerticalTabStripStateControllerTest : public testing::Test {
 };
 
 TEST_F(VerticalTabStripStateControllerTest, Initial) {
-  EXPECT_FALSE(controller()->ShouldDisplayVerticalTabs());
+  EXPECT_TRUE(controller()->ShouldDisplayVerticalTabs());
   EXPECT_FALSE(controller()->IsCollapsed());
   EXPECT_EQ(kVerticalTabStripDefaultUncollapsedWidth,
             controller()->GetUncollapsedWidth());
@@ -120,6 +120,8 @@ TEST_F(VerticalTabStripStateControllerTest, FeatureDisabled) {
 
 TEST_F(VerticalTabStripStateControllerTest, VerticalTabsEnabledFirstTime) {
   base::UserActionTester user_action_tester;
+  pref_service()->SetBoolean(prefs::kVerticalTabsEnabled, false);
+  pref_service()->SetBoolean(prefs::kVerticalTabsEnabledFirstTime, false);
   ASSERT_FALSE(
       pref_service()->GetBoolean(prefs::kVerticalTabsEnabledFirstTime));
   ASSERT_EQ(0,
